core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -232,6 +232,9 @@ class KafkaApis(val requestChannel: RequestChannel,
         case ApiKeys.DESCRIBE_QUORUM => requestHelper.closeConnection(request, util.Collections.emptyMap())
         case ApiKeys.FETCH_SNAPSHOT => requestHelper.closeConnection(request, util.Collections.emptyMap())
 
+        // Handle requests that should have been sent to the KIP-500 controller.
```

Comment:
nit: this applies to all of the Raft apis above as well

Reply:
Ack.